virt/virt-2-4-release-notes.adoc

```diff
@@ -8,56 +8,105 @@ toc::[]
+
+[id="virt-2-4-storage-new"]
+=== Storage
+//Placeholder for new content.
+
+//CNV-5274 - Customize accessMode and volumeMode when creating VM disks.
+
+//CNV-5422 - CNV support for Ceph-based OCS 4.4.
+
+//CNV-5423 - CDI can now import, clone, and upload virtual machine disks into namespaces that are restricted by resource quotas. Administrators can configure the resource limits applied to CDI worker Pods.
+
+//CNV-5424 - virtctl can now use a DataVolume when uploading virtual machine disks to the cluster which helps prevent virtual machines from being inadvertently started before an upload has completed.
 
-* Managing virtual machines is simpler and more efficient due to improvements
-in design and workflow. You can now:
-** Run the virtual machine wizard with less navigation. The wizard now uses
-a comprehensive in-page style and includes a review page for confirming
-configuration details before submission.
-** Import a single VMware virtual machine with less navigation.
-** Edit virtual machine templates as well as virtual machine configurations.
-** Monitor health of virtual machine-backed services as you would for
-Pod-based services.
-** Enable persistent local storage for virtual machine images.
-** Add, edit, and view virtual CD-ROM devices attached to a virtual machine.
-** Add and view network attachment definitions with a graphical editor.
-
-== Resolved issues
-
-* Previously, when you added a disk to a virtual machine via the *Disks* tab in
-the web console, the added disk had a `Filesystem` volumeMode regardless of the
-volumeMode set in the `kubevirt-storage-class-default` ConfigMap. This issue has been fixed.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1753688[*BZ#1753688*])
-
-* Previously, when navigating to the *Virtual Machines Console* tab,
-sometimes no content was displayed. This issue has been fixed.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1753606[*BZ#1753606*])
-
-* Previously, attempting to list all instances of the {VirtProductName} operator
-from a browser resulted in a 404 (page not found) error.
-This issue has been fixed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1757526[*BZ#1757526*])
-
-* Previously, if a virtual machine used guaranteed CPUs, it was not scheduled
-because the label `cpumanager=true` was not automatically set on nodes.
-This issue has been fixed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1718944[*BZ#1718944*])
+//CNV-5421 - DataVolumes have been enhanced with conditions that make it easier to understand the state of VM disk import, clone, and upload.
 
+[id="virt-2-4-web-new"]
+=== Web console
+//Placeholder for new content.
+
+//CNV-5275 - Update template designs - virtual machines and virtual machine templates merged under one menu called Virtualization.
+
+//CNV-5272 - Configure scheduling directly from CNV UI.
+
+//CNV-5273 - Attach a config map / secret / service account to a VM from the UI.
+
+== Notable technical changes
```

Comment:
It can be added later but this should have an anchor ID (probably [id="virt-2-4-changes"] )

Reply:
Added the anchor ID.